feed.go

```diff
@@ -152,21 +151,16 @@ func (feed *ArtifactsFeed) GetLatestArchive(series string, options BuildOptions)
 		return "", errors.Wrapf(err, "problem fetching download information for series '%s'", series)
 	}
 
-	// if it's a dev version: then the branch name is in the file
-	// name, and we just take the latest from master
-	seriesNum, err := strconv.Atoi(string(series[2]))
+	isDev, err := version.isDevelopmentSeries()
 	if err != nil {
-		// this should be unreachable, because invalid
-		// versions won't have yielded results from the feed
-		// op
-		return "", errors.Wrapf(err, "version specification is invalid")
+		return "", errors.Wrap(err, "problem determining version type")
 	}
 
-	if seriesNum%2 == 1 {
+	// If it's a development series, we just replace the version with the word latest.
+	// Otherwise the branch name is in the file name, and we take the latest from master.
```

Comment:
"Take the latest from the series release" - only the development series has to take the latest from master (at leastfor now).